File: source/api/tokens/create/_create.js
omise.tokens.create({
  'card':{
    'name': 'JOHN DOE',
    'city': 'Bangkok',
    'postal_code': 10320,
    'number': '4242424242424242',
    'expiration_month': 2,
    'expiration_year': 2017,
    'security_code': 123
  }
}, function(error, token) {
  /* Response. */
});

File: source/v20140727/api/tokens/create/_create.cs
var card = new CardCreateInfo ();
card.Name = "Somchai Prasert";
card.Number = "4242424242424242";
card.ExpirationMonth = 10;
card.ExpirationYear = 2018;
card.City = "Bangkok";
card.PostalCode = "10320";
card.SecurityCode = "123";

var token = new TokenInfo ();
token.Card = card;

var resultToken = client.TokenService.CreateToken (token);
